### DISCUSSION TITLE
Override config options from the command line

### DISCUSSION THREAD

**Discussion by inducer** [helix-editor/helix#9612] [Q&A]
It'd be nice if config values could be overridden from the command line. My use case is that I'd like to change my theme based on time of day/whether dark mode is enabled in my DE. Some other approaches that might help me achieve the same thing:

- Execute commands from the command line (i.e. `:set` for me)
- Allow environment variable expansion in the configuration

Either of the two feels like a bigger can of worms than the config overrides.

**Comment by kirawi** [comment]
See https://github.com/helix-editor/helix/issues/2158

Helix will reload the config if it receives the USR1 signal.
